### PR TITLE
[gitops](frontend): bump staging and training to v2.1.0-RC

### PR DESCRIPTION
**Pull request by sebastien-comeau**
### Description

As per title.

### CODE REVIEW

File: gitops/overlays/staging/patches/deployments.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ spec:
     spec:
       containers:
         - name: canada-dental-care-plan-frontend
-          image: dtsrhpdevscedacr.azurecr.io/canada-dental-care-plan/canada-dental-care-plan-frontend:0.0.0-cdde95df-00572
+          image: dtsrhpdevscedacr.azurecr.io/canada-dental-care-plan/canada-dental-care-plan-frontend:2.1.0-RC
```
Comment (gregory-j-baker) [inline gitops/overlays/staging/patches/deployments.yaml:13]:
```suggestion
          image: dtsrhpdevscedacr.azurecr.io/canada-dental-care-plan/canada-dental-care-plan-frontend:2.1.0-RC092
```

File: gitops/overlays/training/patches/deployments.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ spec:
     spec:
       containers:
         - name: canada-dental-care-plan-frontend
-          image: dtsrhpdevscedacr.azurecr.io/canada-dental-care-plan/canada-dental-care-plan-frontend:0.0.0-cdde95df-00572
+          image: dtsrhpdevscedacr.azurecr.io/canada-dental-care-plan/canada-dental-care-plan-frontend:2.1.0-RC
```
Comment (gregory-j-baker) [inline gitops/overlays/training/patches/deployments.yaml:13]:
```suggestion
          image: dtsrhpdevscedacr.azurecr.io/canada-dental-care-plan/canada-dental-care-plan-frontend:2.1.0-RC092
```

**Review by gregory-j-baker — APPROVED** [review]

File: gitops/overlays/staging/patches/deployments.yaml (hunk 1 above)
Comment (sebastien-comeau) [reply gitops/overlays/staging/patches/deployments.yaml]: Fixed

File: gitops/overlays/training/patches/deployments.yaml (hunk 2 above)
Comment (sebastien-comeau) [reply gitops/overlays/training/patches/deployments.yaml]: Fixed

**Pull request merged**